### DISCUSSION TITLE
Reverse scrolling on mouse but keep on trackpad

### DISCUSSION THREAD

**Discussion by ushi002** [linearmouse/linearmouse#273] [Ideas]
Would it be possible to turn-on reverse vertical scrolling on mouse but keep the "natural" scrolling on trackpad? I am asking as I feel more comfortable and "natural" to scroll on mouse like in windows (by turning on the reverse scrolling feature), however I don't like to have the trackpad scrolling reversed at all :(

Not sure if it is even possible to reverse scrolling on mouse and keep the original setting on trackpad.

**Comment by lujjjh** [comment]
Sure! That's why LinearMouse exists. You can configure per-device settings.

You may want to refer to [#210](https://github.com/linearmouse/linearmouse/issues/210#issuecomment-1199010477).

**Reply by ushi002** [reply]
I am sorry I have not noticed that little bar showing the active device (and as it is actually switching between trackpad/mouse as I touch one or the other device). Great job! Thanks a lot!
